### PR TITLE
WT-12441 Fix format in the documentation related to compaction

### PR DESCRIPTION
**Pull request by etienneptl**

### CODE REVIEW

File: src/docs/arch-compact.dox [hunk 1]
```diff
@@ -21,15 +21,15 @@ that there are as many blocks as possible on the file's "available for reuse" li
...
 Once enough free space is available, one of the two following conditions has to be met for compaction to proceed:
 
-    - At least 20% of the available space lies within the first 80% of the file (compaction will
+- At least 20% of the available space lies within the first 80% of the file (compaction will
 then try to rewrite the last 20% of the file).
-    - At least 10% of the total file is available in the first 90% of the file (compaction will then
+- At least 10% of the total file is available in the first 90% of the file (compaction will then
 attempt to work on the last 10% of the file).
 
 Depending on the scenario, compaction the object is done 20% or 10% at a time, that is, compaction tries to move blocks from the last 20% or 10% of the
```
Comment (Sean04) [inline src/docs/arch-compact.dox:35]:
```suggestion
Depending on the scenario, compaction is done on 20% or 10% at a time, that is, compaction tries to move blocks from the last 20% or 10% of the
```

File: src/docs/arch-compact.dox [hunk 2]
```diff
@@ -75,13 +75,14 @@ with the compaction process.
...
-keep track of tables where compaction is successful. Using those statistics, the service focuses on
-tables where compaction is likely to reclaim space and avoids unnecessary attempts.
+Background compaction retrieves the files to compact by walking the metadata file. To limit disruption,
+background compaction reduces the number of checkpoints by skipping the first one described in the
+section above. Furthermore, background compaction gathers statistics over time to keep track of tables
+where compaction is successful. Using those statistics, the service focuses on tables where compaction
+is likely to reclaim space and avoids unnecessary attempts.
 
 Background compaction can be enabled, configured, and disabled through the same API \c WT_SESSION::compact.
 When disabled, background compaction goes off as soon as possible and when re-enabled, it starts
```
Comment (Sean04) [inline src/docs/arch-compact.dox:88]: goes off -> turns off

Comment (etienneptl) [reply src/docs/arch-compact.dox]: 0ed33d3179e6c1e559a28974992cc153f2944907

**Review by Sean04 — APPROVED** [review]
LGTM!

**Pull request merged**